### PR TITLE
NAS-117275 / 13.0 / NAS-117275: Logging into WebUI immediately after failover results in …

### PR DESCRIPTION
**Pull request by undsoft**
…both nodes rebooting

### CODE REVIEW

File: src/app/app.component.ts [hunk 1]
```diff
@@ -79,7 +79,9 @@ export class AppComponent {
     router.events.subscribe((s) => {
```
Comment (RehanY147) [inline src/app/app.component.ts:79]: Should rename `s`. Ambiguous.

**Review by RehanY147 — COMMENTED** [review]
Need testing instructions and VM to test on

File: src/app/app.component.ts (hunk 1 above)
Comment (undsoft) [reply src/app/app.component.ts]: Fair enough :)

**Comment by undsoft** [comment]
Testing instructions are: connect to the machine in ticket, click Initiate Failover on main dashboard, login. The machine should not reboot a second time.

**Review by bvasilenko — APPROVED** [review]
Looks good to me. However it needs more testing with the HA enabled setups

**Pull request merged**